src/terminal/terminal.rs

```diff
@@ -568,6 +568,65 @@ where
     }
 }
 
+#[cfg(feature = "crossterm")]
+impl Terminal<CrosstermBackend<Stdout>> {
+    /// Initializes the terminal with reasonable defaults for most applications.
+    ///
+    /// The Terminal is initialized with the following settings:
+    ///
+    /// - Stdout as the output
+    /// - Crossterm backend
+    /// - Raw mode enabled
+    /// - Alternate screen buffer enabled
+    /// - A panic hook that restores the terminal before panicking
+    ///
+    /// For more control over the terminal initialization, use [`Terminal::new`] or
+    /// [`Terminal::with_options`].
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// let terminal = ratatui::Terminal::init()?;
+    /// # std::io::Result::Ok(())
+    /// ```
+    pub fn init() -> io::Result<Terminal<CrosstermBackend<Stdout>>> {
+        Self::set_panic_hook();
```

Comment:
Just thinking out loud, are there instances where a user is going to _not_ want this?

Reply:
I we decide to go with this maybe we can also add e.g. `init_with_options(logging: bool)` or something similar in the future as well.

Reply:
The current approach is the verbose approach. This PR (or better #1180) introduces quick, minimal setup methods to have less setup needed for the common usage. If that's not fitting, the already existing more verbose code can still be used.